Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV["TERM"] = "xterm-256color"
+ENV["GOPATH"] = "/go"
+ENV["LC_ALL"] = "en_US.UTF-8"
+
+Vagrant.configure("2") do |config|
+  # config.vm.box = "fedora/25-cloud-base"
+  # config.vm.provision "shell", inline: "dnf install -y go git docker"
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.provision "shell", inline: "DEBIAN_FRONTEND=noninteractive apt-get install -y golang git docker.io systemd-container tmux"
+
+  # config.vbguest.auto_update = true
+
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+  config.vm.synced_folder ".", "/go/src/github.com/kinvolk/kubeadm-nspawn", create: true
```

Comment:
It does not build in the VM:

```
ubuntu@ubuntu-zesty:/go/src/github.com/kinvolk/kubeadm-nspawn$ make
go build -o cni-noop ./cmd/cni-noop
go build -o cnispawn ./cmd/cnispawn
go build -o nspawn-runc ./cmd/nspawn-runc
go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
pkg/nspawntool/node.go:26:2: cannot find package "k8s.io/kubernetes/pkg/util/procfs" in any of:
	/go/src/github.com/kinvolk/kubeadm-nspawn/vendor/k8s.io/kubernetes/pkg/util/procfs (vendor tree)
	/usr/lib/go-1.7/src/k8s.io/kubernetes/pkg/util/procfs (from $GOROOT)
	/go/src/k8s.io/kubernetes/pkg/util/procfs (from $GOPATH)
Makefile:5: recipe for target 'all' failed
make: *** [all] Error 1
```

Maybe add more synced_folder for the dependencies?